src/EditorFeatures/Core/Implementation/Classification/ClassificationTypeDefinitions.cs

```diff
@@ -27,6 +27,11 @@ internal sealed class ClassificationTypeDefinitions
         [Name(ClassificationTypeNames.VerbatimStringLiteral)]
         [BaseDefinition(PredefinedClassificationTypeNames.FormalLanguage)]
         internal readonly ClassificationTypeDefinition StringVerbatimTypeDefinition;
+
+        [Export]
+        [Name(ClassificationTypeNames.StringEscapeCharacter)]
+        [BaseDefinition(PredefinedClassificationTypeNames.FormalLanguage)]
+        internal readonly ClassificationTypeDefinition StringEscapeCharacterTypeDefinition;
```

Comment:
>StringEscapeCharacterTypeDefinition [](start = 55, length = 35)

This change (with the export) is beyond my current understanding. I'll skip this part since Sam already reviewed and he knows this stuff :-)